server/src/server.ts

```diff
@@ -103,6 +84,9 @@ connection.onInitialized(() => {
 			connection.console.log('Workspace folder change event received.');
 		});
 	}
+	connection.console.log(
+		'' + cloudify.init().then(result => {return result})
```

Comment:
Is `connection.console.log` working with Promises? 🤔

Reply:
I've actually got rid of it and moved it into another location.